File: Modul-4/Soal-4/PRAK404-2410817220023-CLARISSADHEAALLISYA.c
#include <stdio.h>

int main () {
    int pilihan;
    float nilai1, nilai2, hasil;

    while (1) {
        printf("\nPilih program\n");
        printf("1. Penjumlahan\n");
        printf("2. Pengurangan\n");
        printf("3. Perkalian\n");
        printf("4. Pembagian\n");
        printf("5. Exit\n");
        printf("Masukkan pilihan : ");
        scanf("%d", &pilihan);

        if (pilihan == 5) {
            printf("Terimakasih, telah menggunakan kalkulator CLARISSADHEAALLISYA\n");
            break;
        }
        if (pilihan < 1 || pilihan > 5) {
            printf("Input anda salah, silahkan coba lagi\n");
            continue;
        }
        printf("Masukkan nilai pertama : ");
        scanf("%f", &nilai1);
        printf("Masukkan nilai kedua : ");
        scanf("%f", &nilai2);
        switch (pilihan) {
            case 1:
                hasil = nilai1 + nilai2;
                printf("Hasil penjumlahan antara %.2f dengan %.2f adalah %.2f\n", nilai1, nilai2, hasil);
                break;
            case 2:
                hasil = nilai1 - nilai2;
                printf("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f\n", nilai1, nilai2, hasil);
                break;
            case 3:
                hasil = nilai1 * nilai2;
                printf("Hasil perkalian antara %.2f dengan %.2f adalah %.2f\n",nilai1, nilai2, hasil);
                break;
            case 4:
                hasil = nilai1 / nilai2;
                printf("Hasil pembagian antara %.2f dengan %.2f adalah %.2f\n", nilai1, nilai2, hasil);
                break;
        }
    }
    return 0;
}

File: Modul-4/Soal-4/PRAK404-2410817220023-CLARISSADHEAALLISYA.py
def kalkulator():
    while True:
        print("\nPilih program")
        print("1. Penjumlahan")
        print("2. Pengurangan")
        print("3. Perkalian")
        print("4. Pembagian")
        print("5. Exit")
        pilihan = input("Masukkan Pilihan : ")
        
        if pilihan == "5" :
            print("Terimakasih, telah menggunakan kalkulator CLARISSADHEAALLISYA\n")
            break
        
        if pilihan not in ["1","2","3","4"] :
            print("Input anda salah, silahkan coba lagi")
            continue
        
        nilai1 = float(input("Masukkan nilai pertama : "))
        nilai2 = float(input("Masukkan nilai kedua : "))
        
        if pilihan == "1" :
            hasil = nilai1 + nilai2
            operasi = "penjumlahan"
        elif pilihan == "2" :
            hasil = nilai1 - nilai2
            operasi = "pengurangan"
        elif pilihan == "3" :
            hasil = nilai1 * nilai2
            operasi = "perkalian"
        elif pilihan == "4" :
            hasil = nilai1 / nilai2
            operasi = "pembagian"
        print(f"Hasil {operasi} antara {nilai1: .2f} dengan {nilai2: .2f} adalah {hasil: .2f}")

kalkulator()